hugo/layouts/shortcodes/citation_block.html

```diff
@@ -0,0 +1,18 @@
+{{ $year := now.Year }}
+{{ $date := now.Format "January 2, 2006" }}
+
+<!-- get the version number if prod build or set to X.X.X. if not -->
+{{ $version := partial "GetVersion.html" }}
+{{ $portalVersion := "X.X.X" }}
+{{ if $version.tag }}
+    {{ $portalVersion = index $version "tag" }}
+    {{ $portalVersion = replace $portalVersion "v" "" }}
+{{ end }}
+
+<!-- can't pipe the replace command, hence multiline-->
```

Comment:
`strings.ReplaceRE` can be piped though, if you are tempted 🤷🏻‍♂️

Reply:
Haha, I put that comment because I thought you'd ask otherwise, good to know there is a way around it, will include it shortly!

Reply:
You can put `<!-- noqå -->` 🤣

Reply:
```{{ $content = $content | strings.ReplaceRE "DATE_ACCESSED" $date -1 | strings.ReplaceRE "YEAR" $year -1 | strings.ReplaceRE "VERSION_NUMBER" $portalVersion -1 }}``` works indeed! 
But I will keep it multline as I think the line is too long otherwise

Reply:
Problem solved:
```
{{ $content = $content | 
strings.ReplaceRE "DATE_ACCESSED" $date -1 | 
strings.ReplaceRE "YEAR" $year -1 | 
strings.ReplaceRE "VERSION_NUMBER" $portalVersion -1 
}}
```